### PR TITLE
captrs#4 - added geometry fn on DXGIManager

### PR DESCRIPTION
**Pull request by pballandras**
I followed instructions here: https://github.com/bryal/captrs/issues/4#issuecomment-676503941

This will be part 1 of the fix. this isn't the prettiest implementation, if you could help me prettify it, that would be very nice. In the meantime, I'll go open the part 2 PR on the captrs repo!

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -265,6 +265,17 @@ impl DXGIManager {
         }
     }
 
+    pub fn geometry(&mut self) -> (usize, usize) {
```
Comment (bryal) [inline src/lib.rs]: I think `geometry()` could be a `&self` method.

File: src/lib.rs [hunk 2]
```diff
@@ -265,6 +265,17 @@ impl DXGIManager {
...
+        let output_desc = self.duplicated_output.as_mut().unwrap().get_desc();
```
Comment (bryal) [inline src/lib.rs]: would just require changing this `.as_mut()` to `.as_ref()`

**Review by bryal — CHANGES_REQUESTED** [review]
Thank you for the contribution! This almost looks good to go to me. There's just one thing: I don't think `geometry()` has to be mutable. From looking at the issue this relates to, I realize this is probably because you looked at the code I wrote, and I believe now that my code there is `mut` needlessly. Regardless, would you mind trying to make `geometry()` into a `&self` method and see if it compiles?

File: src/lib.rs (hunk 2 above)
Comment (pballandras) [reply src/lib.rs]: See, I didn't know that! :O I'll change that and see if it works so we can merge this bad boy

**Comment by bryal** [comment]
Great! Thanks again!

**Pull request merged**